sdk/src/main/scala/io/horizen/account/state/McAddrOwnershipMsgProcessor.scala

```diff
@@ -38,24 +40,65 @@ case class McAddrOwnershipMsgProcessor(params: NetworkParams) extends NativeSmar
   override val contractAddress: Address = MC_ADDR_OWNERSHIP_SMART_CONTRACT_ADDRESS
   override val contractCode: Array[Byte] = Keccak256.hash("McAddrOwnershipSmartContractCode")
 
-  // ecdsa curve y^2 mod p = (x^3 + 7) mod p
-  val ecParameters: X9ECParameters = SECNamedCurves.getByName("secp256k1")
+  override def init(view: BaseAccountStateView, consensusEpochNumber: Int): Unit = {
+    if (!isForkActive(consensusEpochNumber)) {
+      log.warn(s"Can not perform ${getClass.getName} initialization, fork is not active")
```

Comment:
I would remove the warn (if a sidechain does never activate the fork would end up with 1 useless warn per block)